backend/typescript/rest/camperRoutes.ts

```diff
@@ -105,6 +106,20 @@ camperRouter.post(
   },
 );
 
+camperRouter.put("/waitlist/:waitlistedCamperId", async (req, res) => {
```

Comment:
let's make this a patch request 
```suggestion
camperRouter.patch("/waitlist/:waitlistedCamperId", async (req, res) => {
```